src/examples/mnist/MNIST_SP.cpp

```diff
@@ -58,14 +58,15 @@ class MNIST {
 
 void setup() {
 
-  input.initialize({28, 28});
+  input.initialize({28 * 28});
```

Comment:
why the change from 2D to flat? (28*28)

Reply:
It's easier to work with.  Besides, there is no topology in this example because it uses global inhibition and the radius is larger than the inputs.